hooks/XDR.js

```diff
@@ -0,0 +1,160 @@
+// $IIXDR,C,C,10.7,C,AIRTEMP,A,0.5,D,HEEL,A,-1.-3,D,TRIM,P,1.026,B,BARO,A,A,-4.-3,D,RUDDER*18
+
+const math = require('math-expression-evaluator');
+const fs = require('fs');
+const xdrDictionary = { definitions: [] };
+
+try {
+  // Populate a dictionary
+  const xdrDictPath = require.resolve('xdr-parser-plugin/xdrDict');
```

Comment:
I am not terribly fond of having a configuration file that is loaded using `require.resolve`. I think configuration should be injected from the context that uses the parser. We can support both though if you think this is the way to go, just that we should not log a warning if there is no configuration file.

Reply:
See previous comment regarding `xdr-parser-plugin`